### DISCUSSION TITLE
AAB build file size doubled, IS IT NORMAL?

### DISCUSSION THREAD

**Discussion by flixbeat** [jelte/be.khepri.play.assetdelivery.addressables#73] [Q&A] [answered]
Without addressable the file size is only around 400mb+ after integrating addressable and this plugin the build size is now 900mb+ more than double the size of the previous build.

The question is, is this intentional or am I missing something?

**Comment by flixbeat** [comment, marked as answer]
I was able to determine the cause why this happens.

TLDR: Asset bundles are duplicated in base build.

Scenario:
1. My project only uses PAD and I decided to use addressables and this plugin. When you first imported the addressable plugin to your project, Unity will prompt you to convert all your existing asset bundles that were generated by PAD. https://docs.unity3d.com/Packages/com.unity.addressables@1.1/manual/AddressableAssetsMigrationGuide.html
2. I hit 'Yes' from the prompt and those asset bundles from StreamingAssets folder were automatically converted as Addressable Groups.
3. Once you build the project, you'll notice that the aab build size is doubled.

Solution:
1. Delete all addressable groups that were automatically converted by Unity.
2. Start from scratch and manually put assets that are only needed to addressable groups.
3. Delete all existing asset bundles in your StreamingAssets folder
4. Rebuild
